pkg/probo/document_service.go

```diff
@@ -435,12 +447,26 @@ func (s *DocumentService) SendSigningNotifications(
 					}.Encode(),
 				}
 
+				tmpl, err := template.New("signRequest").Parse(signEmailTemplate)
```

Comment:
We want use the following pattern in a variable on top of the file: `var t = template.Must(template.New("name").Parse("html"))`

This ensure we are not able to build the application if the template has syntax error.